### ISSUE TITLE
05/21: Lewis

### ISSUE THREAD

**Issue by shevajia** [uchicago-computation-workshop/Spring2020#5]
Comment below with questions or thoughts about the reading for this week's workshop.

Please make your comments by Wednesday 11:59 PM, and upvote at least five of your peers' comments on Thursday prior to the workshop. You need to use 'thumbs-up' for your reactions to count towards 'top comments,' but you can use other emojis on top of the thumbs up.

**Comment by policyglot** [comment]
Thank you, Dr. Lewis. for this interesting exploration of links between linguistics, psychology and gender stereotypes. Two questions:

a) While your paper touches on the causal mechanism and the possibility of a third mediating variable, what link does your conclusion hold with the Sapir-Whorf hypothesis? Some authors like Kay and Kempton (1984) concede that casting the 'hypothesis' into an empirically testable one has proven challenging. Nonetheless, to what extent do you think your work can explore its core ideas of linguistic determinism, and how language shapes the way we interpret our world? 

b) In Figure 3 a), you plot the male/career association against the proportion of gendered words for professions. Right at the bottom, I noticed Hindi, which happens to be my native language. From my own experience, Hindi has a large number of female versions for career words, with distinct morphological markers. For example, _dhobi_ is a washerman, and dhoban is a washerwoman. But for more 'modern' professions (with the word most likely imported from English), terms like 'doctor' are indeed gender-neutral. The same may be true of other non-occidental languages in your plot. Could it be possible that the specific set of professions for which gender-specific nouns exist itself varies by the culture and history of each country in the sample? 

Kay, P., & Kempton, W. (1984). What is the Sapir‐Whorf hypothesis?. American anthropologist, 86(1), 65-79

**Comment by ChivLiu** [comment]
Thank you for the presentation! I always consider gender inequality and cultural stereotypes as important fields for education. An interesting fact is that, no matter which background we were from, many teachers and parents have told us that girls perhaps could not learn natural sciences or computational subjects as well as boys. However, the fact is that we would all have the same chance to succeed in a field if we study in the same room. From the figures shown in the paper, I found that Mandarin is relatively a gender-neutral language. Mandarin nowadays has been greatly different from traditional Chinese, and most urban families in China have become very open-minded. On the other hand, traditional Chinese cultures still consider males to be more work-focused and females to be more family-focused. Therefore, I just have an unimportant suggestion for this study. Since the paper said that they studied the most commonly used languages from the countries, it means that the collected terms from those languages perhaps didn't include dialect terms. Hence, I believe including those terms would further improve the accuracy of the model.

**Comment by ydeng117** [comment]
Thank you for the presentation. It is very interesting that our languages can influence our gender ideology in an unconscious way. My first question is how do you distinguish the effect of language on our ideologies from the effect of our ideologies on our languages?
As we learn our languages through experiences, parents, teachers, media, etc, we are socialized into certain gender stereotypes. As a result, we have gendered language patterns that are derived from these stereotypes. Your research seems to present another casual relation: as we are exposed to the gendered language structures, we gain certain gender stereotypes. Hence, there seems to have a "chicken or the egg" causality dilemma: whether our gender stereotype leads to our gendered language structure or our language structure that reflects a certain ideology generates stereotypes? I am curious that how do you solve such a causality problem in your research? 
My second question relates to the concept of "doing gender". As many gender study scholars suggested, we can manage our gender impressions on others. For example, a female coach can display her masculinity by showing her assertiveness and aggression, and a male nurse can also illustrate his femininity in nurturing his patients. How would you imagine the language structure can help people in doing their gender? and what implications do you have for language usage in promoting gender equality?

**Comment by wanitchayap** [comment]
Thank you in advance for your presentation! This is a very interesting topic and I am very excited to hear your talk. I have 2 short questions:

1. In your future direction, you mentioned investigating the degree of exposure needed to form differences in belief as well as this phenomenon in bilingual individuals. I can think of three samples to test these two questions: bilingual adults, bilingual children, and L2 learners. I guess it is possible to find data for adult sample. However, the results might not be completed without the children sample (ex. L2 aldult's learning trajectory and final competency is different from a bilingual child learning the 2 languages), but the children sample might not be as easily obtained as adult sample without in-person experiment. And even with the in-person experiment, it is hard to get enough data to train a reliable embedding model. How would you investigate these questions in children sample using observational study?

2. I think it is really admirable that you pre-registered your study! I know about the concept of pre-registration and that the scientific community tries to encourage researchers to do so. However, I have never heard about it first-handed. Could you explain more about your experience of doing so?

**Comment by bakerwho** [comment]
Thanks in advance for sharing with us.

Your use of word embeddings to infer gender stereotypes is novel and very interesting! I really appreciated the elegant way in which you used context-based word embedding models to examine the associations of gendered words with other words.

My first question is on the already skewed representation of non-male writers and characters in most movies and TV shows. There's a lot of analysis on how predominantly male casts or writing teams tend to misrepresent non-male characters, exaggerating and often oversexualizing traits. I'd guess that the ground-truth stereotype (so to speak) might then be slightly less strong than what you inferred from the Subtitle corpus. Was there any noticeable difference between your results on Wikipedia and the Subtitle corpus?

Secondly, did you study account for the grammatical or usage-driven gendering of the words themselves? For instance, many Sanskrit-root adjectives roughly translating 'great' tend to be adopted as male names (Virat, Vishal), while many for 'gentle' (Saumya, Namrata, Vinita) tend to be female names. In French 'richesse' and 'fortune' are feminine nouns. I'm not sure how this might influence the effect you studied, but was curious about it anyway.

**Comment by luxin-tian** [comment]
Thank you in advance for your presentation. This is quite an innovative perspective to explore potential causal pathways between language and gender stereotypes. I have one concern about the inference of the causal relationship, though might be trivial. Since many organizations and social movements are advocating the usage of gender-neutral language, involving many languages, especially those that are used as official working languages in public sectors or international organizations such as the United Nations. In this case, the language read by people, from their early ages, can be somewhat refined to align with such initiatives. However, such changes can be imbalanced across different groups of people with different levels of education, social status, family backgrounds, etc. If it is the case, there can be significant heterogeneity underlying the causal mechanism, and the influence of such movements on language can be lagged-behind and hardly observable in an immediate manner. How can we further address these problems?

**Comment by hesongrun** [comment]
Thanks for your presentation! This is really an interesting topic. Your research shed light on the role our language plays in molding the gender identity. My question is how do you perform causal inference of language people used on the formation of man-career/woman-family stereotypes? How do you address the endogeneity issue here? It could be the case that people's attitudes toward gender identity cause them to use certain language. So how do you address the problem of reverse causality in this study? Thanks!

**Comment by guohaihao** [comment]
Thanks for your presentation in advance! I am wondering how do you address the possible endogeneity problem here? Thanks a lot!

**Comment by bhargavvader** [comment]
Thank you Molly, it is always exciting to hear an ex-Knowledge Lab member speak!

@policyglot already pointed out the consequences this has for the Sapir-Whorf hypothesis - and @hesongrun and @hihowme have both discussed the issues with endogeneity and how measuring the extent of the Sapir-Whorf-ness of this can become an exercise in futility.

I also notice that there is no mention of this in your paper - is it because the hypothesis isn't really taken very seriously in this kind of more quantitative work, or do you not approach the problem with that in mind? Keith Chen approaches this conversation in his work ([The Effect of Language on Economic Behavior: Evidence from Savings Rates, Health Behaviors, and Retirement Assets](https://www.aeaweb.org/articles?id=10.1257/aer.103.2.690)), and I thought it was a cool way to go about thinking of how language affects our actions (and thoughts).

So basically, my questions is: what do you think of the Sapir-Whorf hypothesis, and is it at all a guiding thought in your research?

**Comment by Yilun0221** [comment]
Thanks for the presentation! I really like this topic! My question is, what is the purpose of using 25 languages? Do you want to reach a conclusion of the whole population in the world? Or do you want to compare different communities in terms of their languages?

**Comment by linghui-wu** [comment]
Thank you for your brilliant work. Gender discrimination has always been my research of interest and I think the employment of content analysis techniques such as word embedding provides me a new angle to investigate the problem.

I have similar questions as those by my peers:

1. My initial thought about the use of subtitles and Wikipedia corpus in study 1a is to strike a balance between formal written and informal spoken languages. The first question of @bakerwho makes me cannot help but think how the two corpus complement each other and deal with the uneven tendency of gender discrimination in different data sources?

2. As mentioned by @luxin-tian, the social movement calling for gender equality in recent years has promoted the usage of gender-neutral language, I wonder whether we can capture such a dynamic in the study?

**Comment by di-Tong** [comment]
Thank you for sharing this interesting work! Could you talk a little bit more on the possible data and designs for studying the further questions you proposed at the end of the paper (how much exposure to the relevant language statistics is sufficient to produce differences in beliefs, and how resilient the learned associations are to other sources of information)?

**Comment by ShuyanHuang** [comment]
Thank you for sharing your work with us! The concluding sentence in your paper stresses the importance of understanding the causal role of language in the formation of gender stereotypes. In this direction, your paper has found the association between gender stereotypes and structure of natural language semantics. But there still seems to be some distance between "association" and "causal role". Language and culture affect each other in their development. I think it would be interesting and important to separate the influences in these two directions.

**Comment by zeyuxu1997** [comment]
Thanks for sharing your work with us. Your work has shown clearly the fact that language plays a significant role on shaping gender stereotypes. My question is whether the dilution of gender association can cause some unexpected changes in gender identity?

**Comment by lulululugagaga** [comment]
Thanks for your presentation. I feel the same that the gendered expressions/representations in languages have great influence on the formation of gendered behaviors/actions, and your work offers systematic evidence and analysis. Your perspective on studying gender stereotypes is really interesting. I look forward to this workshop!

**Comment by vinsonyz** [comment]
Thanks for your presentation. Very interesting topic. Stereotypes together with gender inequality are two important concerns of social scientists. While the preference for STEM might be intrinsic, how would you justify this in your research?

**Comment by skanthan95** [comment]
Thanks for coming to our workshop, and for your fascinating paper! I had two questions about the methods:

(1) I'm interested in hearing more about the subtitle corpus (used in Study 1 to pre-train word embedding models to determine career-gender associations in English).  Which media were ultimately included in the sample, and why? I noticed that the number of English subtitle files and tokens in the OpenSubtitle database were included [here](https://www.duo.uio.no/bitstream/handle/10852/50459/947_Paper.pdf?sequence=4) and [here](https://www.researchgate.net/publication/336506861_subs2vec_Word_embeddings_from_subtitles_in_55_languages), but didn't see information about the distribution of genres or timeframe. Given that language used in movies and shows varies significantly by genre and time, and some genres might contain more gender-stereotyping language than others (e.g., romance films), I'm curious about the proportion of subtitles included by movie/TV show genre and the representativeness of the sample.

(2) In Study 2, languages with more gendered occupation terms had stronger career-gender associations in their language statistics, with no reliable correlation between grammatical gender and implicit association. Looking at the career-related words chosen (anticipated to be associated with men), I noticed that some of them have a female gender in some of the languages sampled (career: "la profesión", in Spanish). While this phenomenon of "opposite" grammatical and stereotypical genders doesn't seem to have bearing on the present results, are there any other groups of gendered words (i.e., beyond "career" and "family") that you might analyze in future iterations of this work? Do you think this dissonance between the assigned grammatical gender and stereotypical gender would be an important consideration there?

**Comment by dhruvalb** [comment]
Thanks for sharing this interesting work with us! I was wondering about the effect of English being the lingua franca of business in many nations. People in many countries may use their native language at home and English for education/work, morphing their implicit biases. Would it be worthwhile to see which is the dominant career language and how that affects implicit associations?

**Comment by rkcatipon** [comment]
Hi Dr. Lewis, thank you so much for coming to speak to our program. I was impressed with how your research not only looks at semantic patterns in English but across many languages. My question was about the gender analysis used for the Filipino language. As I understand Tagalog (the non-standardized dialect of Flipino) is genderless-- through the cross-pollination of Spanish, Chinese, and American colonization has undoubtedly had an effect on the language. Could you talk a bit more about the process of finding gendered occupation terms for Filipino in study A?

**Comment by YuxinNg** [comment]
Thanks for sharing your work. This is a very interesting topic. Your work shows that gender stereotypes are significantly shaped by language. More and more people pay attention to the gender equality, some progress have been made in gender equality. But language does not change so quickly and easily.  Do you think what you found in your paper, the connection between language and gender stereotypes will be weakened in the future. Thanks!

**Comment by liu431** [comment]
Thanks for the talk. I think gender stereotype is an issue we know it exists, but couldn't be changed easily.  Do you have any thoughts on how to change these stereotypes through education?

**Comment by chiayunc** [comment]
Thank you for your wonderful paper. I have a question on whether or not you have any comment on the fact that in Fig.2(a), there does not seem to exist clustering of languages that one might think share more linguistic roots. What do you think this could indicate? A second question I have is regarding the negative effect size. For some of the languages, there are negative effect sizes in implicit and linguistic associations of man-career.  Am I correct to assume that they actually have reversed association rules? what does this tell us?

**Comment by dongchengecon** [comment]
Thanks a lot for your presentation! The correlation between gender stereotypes and language is quite an interesting topic. I am wondering if you could try measuring the language based on phonetics and see how that measurement could impact gender association.

**Comment by minminfly68** [comment]
Thanks for your presentation. It is quite interesting to see the relationship between linguistics and sociology. I am wondering whether can we explore the effect of language change/idioms/dialects on the sociology? Can we observe the more deeper pattern in this perspective? Thanks!

**Comment by fulinguo** [comment]
Thanks very much for your presentation! My question is that if we control other country-level (and/or individual-level) variables in the associational analysis, whether the correlation between language and gender stereotypes will still be significant. Another question is regarding the causal analysis issues. Could you please discuss more about the changes in language and their relationship with possible changes in gender stereotypes? How could we determine whether the changes in language, if any, are exogenous? Thanks!

**Comment by jsgenan** [comment]
Thank you for bringing your work to us! It is very clever to use the difference in grammatical gender to establish the causal link between linguistic bias and gender stereotype. I have two questions:
1. How would you interepret to difference in results between explicit and implicit association score? Explicit is the difference in word-type choice, while implicit score is the difference in response time. I am not sure how the different results from these two scores represent.
2. Have you considered any longitutinal attribute of the corpus to clear up the causal link? For example, how does the association score evolve across time? I would imagine there being a varaiation across different linguistic context settings.

**Comment by cytwill** [comment]
This is an interesting topic and also related to my recent research! The connections between linguistic properties and social norms are always a heated topic in socio-linguistic areas. My questions are the following:

1.  What are the actual meanings of the explicit association and implicit associations in this research context? That is to say, why you think this kind of measurement can be a proxy for people's gender stereotypes? What intentions do you have to differentiate these two types of associations?
2.  You mentioned about the unreliability of the IAT dataset, and suppose that the results would still be effective on a collective level. I am wondering if there are any alternative datasets or making-up plans for this issue? Otherwise, the explanation itself might not be convincing enough to address the data unreliability.
 
Thank you for your presentation~

**Comment by YanjieZhou** [comment]
Thanks very much for your presentation! Gender ideology is undoubtedly a very critical problem and interesting topic for us to focus on. The language based on this can certainly have some impact on others and on the other hand reflect our gender-related opinions. I am very interested in its application in content analysis.

**Comment by SiyuanPengMike** [comment]
Thanks a lot for your interesting and inspiring paper. Gender stereotypes are a topic that all of us are highly concerned about. The results you showed between different languages are also quite impressive. I'm wondering if you could do a time series analysis to enrich your research. By using more data through more extended time, we might find the trend of gender stereotypes in different cultures, is that becoming worse or better. Thanks again for your paper!

**Comment by tonofshell** [comment]
Thanks for sharing your work! I noticed that you had to obtain translations as a part of the project. Did you recruit native speakers to directly assist you and your team, or did you source these translations from somewhere else?  How difficult was it to find other necessary resources for languages you were not familiar with?

**Comment by yongfeilu** [comment]
Thank you for your presentation! My question is how you clarify whether language exerts influence on gender identity or vice versa? What is the method you recommend to solve the issue? Thank you very much!

**Comment by anqi-hu** [comment]
Thank you so much for sharing your work with us. It is amazing that you are studying gender stereotypes through semantics in many languages from various language families. My question is related to your suggestion of the further direction of investigating the gender-related beliefs of individuals that are bilingual and may be exposed to contradictory linguistic conventions. I think that it might be interesting to look at two types of individuals in that regard: 1) those who were born to and raised in bilingual families and 2) those who grew up monolingual but gained proficiency in another language during their education/other life experiences. Do you expect individuals speaking the same two languages from these two groups to have different trajectories in how their gender-related beliefs are shaped?

**Comment by Leahjl** [comment]
Thank you so much for your presentation! It is quite interesting to combine gender stereotypes and lexical forms. And I'm curious about the gender-neutral language, what would be the impact of adopting such neutral words in the modern world?

**Comment by MegicLF** [comment]
Thank you so much for discussing gender stereotypes! In the paper, you mentioned that the data consists of 25 languages from 8 languages families. Could you discuss the reason that you pick these specific languages and is there any obvious difference between each language family?

**Comment by RuoyunTan** [comment]
I think this study is interesting and meaningful in many ways. I have a small question about the IAT. When a participant classifies a word to a key, such as "female/family", it doesn't necessarily mean he/she believes in this way. In other words, how a participant classifies a word "as quickly as possible" may reflect his/her experience, like seeing this combination of words more often, but it may not reflect his/her own beliefs. Would this affect how we view and use the data?

**Comment by Anqi-Zhou** [comment]
Thank you for the presentation in advance! I think it's really an interesting topic. Could you please explain the IAF in details as the qualifying measurement of cultural stereotypes at an individual level? Can slower reaction time actually reflect the existence of cultural stereotype? What if there exists individual difference?

**Comment by ruixili** [comment]
Thank you for the presentation! This is such a great topic. My question is, under the globalization situation, many languages have started to integrate with words in foreign languages. Do you think that this integration might lead to homogenous cultural stereotypes? What's more, since we are no longer define gender as just two categories, how would this affect your study?

**Comment by Yawei-Li** [comment]
Thank you for your presentation in advance. In my previous understandings, it seems that the gender-based linguistic usage was the result of the stereotypes, yet your work seems to suggest the opposite. How does your work address the mutual causation?

**Comment by sunying2018** [comment]
Thanks for your presentation! In the result of implicit Male-Career association and gendered occupation terms, this paper controls for age, gender, and block order.  I am wondering if the educational background should be also taken into control since education majors may have an important influence on their occupation choice?

**Comment by SixueLiu96** [comment]
Thanks for sharing such great work! I wonder how you deal with some potential endogenous problem in you analysis part? Because for me, this seems to cause some bias in the result.

**Comment by lyl010** [comment]
Thanks for coming and this interesting work. I am curious about the Implicit Association Tests mentioned in discussion, and could you please introduce more details to us? Thank you!

**Comment by harryx113** [comment]
Thank you for the presentation! I'm recently doing a related research on how genders are represented in races. For instance, African Americans are viewed more masculine and Asians are viewed more feminine with respect to white people. I wonder what you think of the interaction between gender and race in the context of language?

**Comment by yutianlai** [comment]
Thanks for coming! Since you are studying language and gender, I'm wondering what linguistic tools do you find particularly useful in tackling such social science problems.

**Comment by heathercchen** [comment]
Thank you for your presentation in advance! Your research topic is really intriguing and interests me a lot. My question is can you obtain casual relationships using the data and results from your analysis? It is the traditional gender norm that enhances what people speak or it is what people speak that reflects gender discrimination?

**Comment by nt546** [comment]
Thanks for sharing your work on the relationship between language and gender stereotypes. In the presence of digitization of books and audiobooks, I was wondering about the feasibility of modifying classic works of literature to reflect gender neutrality.

**Comment by sanittawan** [comment]
Thank you for presenting your research to us. I personally found the paper to be very interesting. I am curious about this particular idea that you mention: 

> For example, if a bilingual individual is exposed to conflicting gender associations in two languages, is the net-effect a combination of the two sources of information or does it dynamically vary with the linguistic context of a given interaction 

What would be your conjecture on this? Will there be a reconciliation process for a truly bilingual speaker? What do you think would be a possible effect of a person who is native to both language with and without a grammatical gender system?

**Comment by Panyw97** [comment]
Thank you for your presentation! It's an interesting topic and also a good fit for computational social science. As social media is developing rapidly, we can get more data on people's thoughts through the news, their posts, etc.

**Comment by weijiexu-charlie** [comment]
Thanks for your presentation. I'm curious about how to tell the direction of the causal pathway in your research. That is, how could you know whether it is the gender markers in linguistics system that lead to people's cultural stereotype or it is people's cultural stereotype that leads to the statistics of gender markers you observe in the linguistics system.

**Comment by WMhYang** [comment]
Thank you very much for your presentation. Language can be the "original sin" of gender inequality since people learn to speak their native language as early as two years old as you mentioned in your paper. However, language is also evolving over time, so the gender associations in a language may also follow a "learning curve". Hence, I wonder if we could build a dynamic model to examine the change in gender associations in languages to better illustrate this pattern, especially in the time and age when languages are heavily affected by social media and popular trends.

**Comment by hanjiaxu** [comment]
Thanks for sharing your research! Certain languages are not gendered (such as Chinese? even if the written characters of she/he are the same, the pronunciations are the same), but these cultures are not necessarily free from stereotypes or gender stereotypes are manifested in more implicit ways, and I am wondering how do you deal with these type of ungendered languages in your research?

**Comment by JuneZzj** [comment]
Thank you for presenting. Your study is  very impressive and I am really interested in the part about word emdedding part in the paper. You also mentioned the cultural stereotype are generated through experience. Could you explain more about how to justify the group-difference are associated with the strength of gender associations encoded in the statistics of different languages? Thank you.

**Comment by keertanavc** [comment]
Thank you for presenting! I noticed that the R-square values of most graphs are low. While I understand that as a rule of thumb, it is generally low for social science data, I was wondering if you considered adding additional covariates (like GDP, constitutional rights for women) for a better fit?

**Comment by jtschoi** [comment]
Thank you in advance for your presentation. I agree with your assessment that correlational approach is an important first step to establishing a more deeply-rooted causal problem that may be present. However, I was wondering whether (if considering the data in a time-series sense) there could be feedback between the implicit and linguistic associations; if so, it could be very difficult to disentangle the causal relationship from a cross-sectional dataset you have at hand. Are there multiple iterations of the data collection that allows us to form a panel study? I apologize in advance if I have missed that part from reading your study.

**Comment by timqzhang** [comment]
Thank you for the presentation. I have the similar question on the purpose of using 25 languages in the paper, and how to control some conditions given the difference in the language patterns and potential biases when translating the different languages?

**Comment by ziwnchen** [comment]
Thanks for the presentation! Studying gender stereotypes in multiple cultural settings is super interesting! I am particularly interested in two points that I feel very thought-provoking in this paper: (1) language may not merely "reflect" existing stereotypes, but also shape and reinforce the culture. How could we estimate the shaping effect when investigating the causal relationship? (2) the concepts of "deep" and "shallow" embedding are super cool! Is it possible to systematically measure such hierarchies using other embedding methods (e.g., hierarchical/hyperbolic embedding).

**Comment by Yiqing-Zh** [comment]
Thank you for your presentation in advance! It is an interesting topic to study the relationship between gender inequality and language stereotypes. I am wondering how to disentangle the causal role of language in the formation of gender stereotypes since there are many potential factors influencing both of them.

**Comment by PAHADRIANUS** [comment]
Thank you for sharing this highly innovative result of yours. I am definitely quite convinced that languages' distributional structure plays a role in shaping the gender stereotypes, homogeneously across various racial and linguistic groups. But just as many others pointed out, I am also concerned with to what degree language can be represented as a singular factor affecting the creation of the stereotyping. One can hardly imagine that, being an inseparable component of a society's cultural, ethical and stratification traditions, language manages to exert such an impact on its own. I think it should be possible and desirable to explore the reciprocal relationship between stereotyping and language evolution.

**Comment by HaowenShang** [comment]
Thanks for your presentation! I see the R-square in the study is not high. Could you explain what’s the criterial for a good R-square and is there any way to improve the R-square? Thanks!

**Comment by ShanglunLi** [comment]
Thanks for providing such an interesting paper to read! Why did you use 25 languages? Is it for the general conclusion?

**Comment by goldengua** [comment]
Thanks for your presentation! I was wondering how your model would deal with the grammatical gender? Does languages such as German which distinguish noun genders would be differently represented from gender neutral nouns such as English?

**Comment by bjcliang-uchi** [comment]
Thank you very much for this presentation. Like many of the other questions have pointed out, there seems to be a grammatical gender that, while indeed represent certain level of gender stereotype--complicates the causal inference between language and mental biases. To this end, I am wondering if bilinguals who adopt their second language after certain age cutoffs can show increase/ decrease in gender biases. You proposed this type of research at the end, and I am very curious to know if there has been any result.

**Comment by wu-yt** [comment]
Thank you for your presentation! My question is how do you think your research will create influence in the current workplace and society?

**Comment by nwrim** [comment]
Thank you for your presentation! If language structure is indeed related to gender stereotypes, what could be a way to decrease this effect? Language structures are accumulated over thousands of years and it will be very hard and take a long time to change the structure itself (i.e. We would not be able to make French drop its gender-specific noun/pronoun/verbs). What could be a step to ameliorate this relationship between language and stereotype?

**Comment by adarshmathew** [comment]
Thank you for your presentation, Dr. Lewis! This is really interesting work. As some of my colleagues have pointed out in this thread, I'm curious about how you're accounting for an imbalance in the production of your corpora -- the subtitle corpora will be dominated by male writers, and Wikipedia might have a similar slant too. Working with the assumption that we reproduce our biases in world construction/framing and writing, how would such an imbalance affect your results?

Also, any and all questions you answer about how to think about the Sapir-Whorf hypothesis when trying to draw inferences with language data would be greatly helpful.

**Comment by KenChenCompEcon** [comment]
Thank you for presenting here. The lexical co-occurrences have always been a critical issue that helps shape the gender stereotypes and sometimes define the realm of jobs that men and women can do. I do look forward to hearing your comments on this and am especially interested in your embedding models on measure the biases in languages.

**Comment by boyangqu** [comment]
Thanks for your presentation. How did you select the languages you use, and in the process, did you find differences among those languages?

**Comment by ZhouXing-19** [comment]
Thanks for your presentation! I am also curious about the language selection process. Thank you!

**Comment by yalingx** [comment]
Thank you for your presenting. How do you decide which language to use?

**Comment by TianxinZheng** [comment]
Thanks for your presentation.
